ver/us/undefined_syms.txt

```diff
@@ -67,6 +67,14 @@ D_7012BC11 = 0x7012BC11;
 D_80026AC7 = 0x80026AC7;
 D_8004AA85 = 0x8004AA85;
 
+// RSP Microcodes
+rspbootTextStart = 0xB0000B70;
+rspbootTextEnd = 0xB0000C70;
```

Comment:
can possibly do this with ld_addrs.h?

Reply:
what u mean? i can't put those in ld_addrs.h